### PR TITLE
adds comments for each article

### PR DESCRIPTION
**Pull request by ConnorK519**

### CODE REVIEW

File: src/components/CommentList.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+export const CommentsList = ({ keyId }) => {
+  const [newComments, setNewComments] = useState([]);
+
+  useEffect(() => {
+    getArticleComments(keyId).then(({ data: { comments } }) => {
+      setNewComments(comments);
+    });
+  }, []);
+
+  return (
```
Comment (NRMSMITH) [inline src/components/CommentList.jsx:18]: Waht would you return if there were no comments?

File: src/api.js [hunk 2]
```diff
@@ -11,3 +11,7 @@ export const getArticleInfo = () => {
 export const getArticle = (key) => {
   return newsApi.get(`/articles/${key}`);
 };
+
+export const getArticleComments = (key) => {
+  return newsApi.get(`/articles/${key}/comments`);
```
Comment (NRMSMITH) [inline src/api.js]: I think perhaps you should be finishing the promise here and _then_ returning the data formatted, rather than returning the promise.

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
A couple of things to consider :)

**Review by itskatherine — APPROVED** [review]
Nice stuff Connor, it's good to merge ⭐

**Pull request merged**